src/seamless_communication/cli/m4t/finetune/dataloader.py

```diff
@@ -191,18 +191,24 @@ def _batch_tensors(self, tensors: List[Tensor], pad_value: Any) -> Tensor:
         return torch.stack([tensor for tensor in padded_tensors], dim=0)
 
     def _is_long_src_audio(self, sample: LangPairSample) -> bool:
-        wav, sample_rate = torchaudio.load(sample.source.audio_local_path)
-        length_s: float = max(wav.shape) / sample_rate
-        return length_s > self.batching_config.max_audio_length_sec
+        # HACK:: causes errored audios to be excluded but this is difficult to follow
+        try:
+            wav, sample_rate = torchaudio.load(sample.source.audio_local_path)
+            length_s: float = max(wav.shape) / sample_rate
+            return length_s > self.batching_config.max_audio_length_sec
+        except:
```

Comment:
except Exception:
   log exception here (log.exception(f"Failed to load sample path={sample.source.audio_local_path}")

Reply:
Done